src/routes/safe/components/AddressBook/index.tsx

```diff
@@ -244,6 +244,14 @@ const AddressBookTable = (): ReactElement => {
                                 explorerUrl={getExplorerInfo(row[column.id])}
                               />
                             </Block>
+                          ) : column.id == AB_NAME_ID ? (
+                            row[AB_NAME_ID] ? (
+                              row[AB_NAME_ID]
+                            ) : (
+                              `${row[AB_ADDRESS_ID].toString().slice(0, 6)}...${row[AB_ADDRESS_ID].toString().slice(
```

Comment:
We export `textShortener` from `@gnosis.pm/safe-react-components` that will make this easier. [Here](https://github.com/gnosis/safe-react-components/blob/c6781499786bdb969b36ca2690c0d6df5b2c2acb/src/ethereum/EthHashInfo/index.tsx#L143)'s an example of how we use it in our `EthHashInfo` component.